src/screens/Proposals.js

```diff
@@ -138,6 +146,14 @@ const Proposals = React.memo(
                   display={compactMode ? 'icon' : 'label'}
                 />
               )}
+              <div>
+                <DropDown
+                  id="perpage"
+                  items={ENTRIES_PER_PAGE}
+                  selected={selected_entries_per_page}
+                  onChange={updateProposalsPerPage}
+                />
```

Comment:
I think a text inside the dropdown would be nice, to be more intuitive for the user!